src/navigation/OccupancyGridClient.js

```diff
@@ -73,7 +73,9 @@ ROS3D.OccupancyGridClient.prototype.processMessage = function(message){
       // grid is of type ROS3D.SceneNode
       this.currentGrid.unsubscribeTf();
     }
-    this.sceneNode.remove(this.currentGrid);
+    if(this.sceneNode){
+      this.sceneNode.remove(this.currentGrid);
+    }
```

Comment:
This is fixed in https://github.com/RobotWebTools/ros3djs/pull/444. We can close this PR and move Three.js update to a separate PR (there already is one here, https://github.com/RobotWebTools/ros3djs/pull/381, but it will be a breaking change and needs a major version bump, plus we need to add more tests because it is very possible that something that is not currently tested will break)